srcs/class/response.cpp

```diff
@@ -51,11 +51,7 @@ int	Response::send() {
 	raw_response += body;
 
 
-	#ifdef PRINT_REQ
-		std::cout << "<====>" << std::endl;
-		std::cout << raw_response << std::endl;
-		std::cout << "<====>" << std::endl;
-	#endif
+	// std::cout << raw_response << std::endl;
```

Comment:
Pour permettre l'affichage de cette info quand on utilise le mod debug!
```suggestion
	#ifdefine DEBUG
		std::cout << raw_response << std::endl;
	#endif
```